instaclustr/api_client.go

```diff
@@ -297,6 +297,27 @@ func (c *APIClient) DeleteEncryptionKey(keyID string) error {
 	return nil
 }
 
+func (c *APIClient) ReadClusterCredentials(clusterID string) (*ClusterCredentials, error) {
```

Comment:
[Desired, reusability] - could `ReadCluster` be used instead? Accordingly you could just enhance the `Cluster` struct

Reply:
I have updated to reuse the existing `ReadCluster` and `Cluster` struct